diff -ru tbb2017_20161128oss/build/linux.gcc.inc tbb2017_20161128oss_modified/build/linux.gcc.inc
--- tbb2017_20161128oss/build/linux.gcc.inc	2016-11-29 14:24:53.000000000 +0000
+++ tbb2017_20161128oss_modified/build/linux.gcc.inc	2017-11-30 18:20:37.271858410 +0000
@@ -86,6 +86,7 @@
     ITT_NOTIFY = -DDO_ITT_NOTIFY
     CPLUS_FLAGS += -m64 $(RTM_KEY)
     LIB_LINK_FLAGS += -m64
+    LIB_LINK_FLAGS += $(DEFAULTFLAGS)
 endif
 
 ifeq (ia32,$(arch))
